src/setupTests.ts

```diff
@@ -2,4 +2,4 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom';
+import "@testing-library/jest-dom";
```

Comment:
and again, if you don't use test, then just remove all references to the tests, and I would remove node modules responsible for testing-libraries and jest